### PR TITLE
add checks on node LB in support.sh

### PR DESCRIPTION
**Pull request by xinfengliu**
Starting from 18.09, there's a per node LB for each overlay
network, this change adds the checks (ip, bridge, iptables, ipvsadm) on node LB.
This change should not break on older docker versions.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @quadespresso @arkodg PTAL

File: support/support.sh [hunk 1]
```diff
@@ -76,23 +76,27 @@ echo "Overlay network configuration"
...
-    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t filter | grep -v '^$'
-    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t nat | grep -v '^$'
-    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t mangle | grep -v '^$'
-    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPVSADM} -l -n
-    printf "\n"
-    ((networks++))
+
+    for i in "${nspath[@]}"
+    do
+      echo_and_run ${NSENTER} --net=${i} ${IP} -o -4 address show
```
[the remark this answers is not in the document]
Comment (xinfengliu) [reply support/support.sh]: Arko, do you mean `block below` is the codes that runs against container netns ? If yes,  `block below` is not 100% same block, there is no `bridge fdb show` and `iptables` on `filter` table. I don't know why there's a small difference here.

[the remark this answers is not in the document]
Comment (arkodg) [reply support/support.sh]: those commands are not applicable inside the container since the Host is performing L2 and L3 forwarding as well

**Comment by thaJeztah** [comment]
ping @mavenugo @joeabbey good to go?

**Pull request merged**